primer/brand
# Updates typographic weight defaults Heading component + adds <em> support into SectionIntro

> [!Important]  
> PR includes visual and component API breaking changes. 
> Release tag: `next-minor`

🔗 [Read the rendered changeset for full details](https://github.com/primer/brand/blob/rezrah/add-emphasis-support/.changeset/fair-rocks-return.md) (To be shared with partner teams ahead of a potential release)

🔗  [See these changes in Storybook](https://primer-a340064e6f-26139705.drafts.github.io/storybook/?path=/story/components-heading--scale)

--- 

## Summary

Part of https://github.com/github/primer/issues/2495 and https://github.com/github/primer/issues/2504
Required to unblock https://github.com/github/primer/issues/2517

Adds support for emphasized text into SectionIntro and makes sweeping visual alignment changes to all headings. 

This is an interim step towards applying global heading and text updates at the Primitives layer. This should help make diffing those PR's easier, by removing the `weight` changes from the list of expected changes.

As this will be a visual breaking change, this will first be merged into the `next-minor` branch (when available) and shared with partner teams to gauge impact ahead of a GA release.


## List of notable changes:



- **added** `<em>` composability into SectionIntro
- **updated** SectionIntro font size to be closer to expected 40px (site refactor). Previously  64px, now 48px.
- **updated** All headings now use either bold (size 1) or semibold (2-6) by default
- **removes** `size` prop from FAQ.Heading as this was introduced before `size` was available in Heading. 

## What should reviewers focus on?

- Thoughts on breaking changes
- Anything look out of the ordinary in the visual diffs
- Changeset review. Missing any details?

## Steps to test:



1. Use the PR diff view on the visual snapshots to compare before/afters
1. Open and review Storybook
1. Open and review the docs, which should match Storybook


## Contributor checklist:


## Reviewer checklist:


## Screenshots:

### New emphasis in SectionIntro component using `<em>`

![Placeholder heading with the first two lines in darker color to the rest of the heading](https://github.com/primer/brand/assets/13340707/14a7ce78-7117-4507-9f87-90f85437e923)
